mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -265,6 +266,18 @@ class ExperimentView extends Component {
     const compareDisabled = Object.keys(this.state.runsSelected).length < 2;
     const deleteDisabled = Object.keys(this.state.runsSelected).length < 1;
     const restoreDisabled = Object.keys(this.state.runsSelected).length < 1;
+    const searchInputHelpTooltipContent = (
+      <div className="search-input-tooltip-content">
+        Click here to learn more about the search syntax.<br/>
+        <a
+          href="https://www.mlflow.org/docs/latest/search-syntax.html"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn more
```

Comment:
Replace 

Click here to learn more about the search syntax.

with 

Search runs within a single experiment or a group of experiments using a simplified version of the SQL ``WHERE`` clause.

It's then obvious that you click the **Learn more** link following the sentence to learn about the syntax.